token/services/ttxdb/db/badger/marshaller.go

```diff
@@ -28,6 +28,11 @@ func MarshalTokenRequest(mr *TokenRequest) ([]byte, error) {
 	return json.Marshal(mr)
 }
 
+// MarshalTransactionEndorseAck marshals a TransactionEndorseAck into a byte array
```

Comment:
The wrapper methods for marshalling make it a bit harder to re-use code. Do we need to have them separate?